README translated to english (#1)
QLife-Inc/sorry: README.md

> # Generic sorry server

Reviewer: ```suggestion
# Plain Sorry Server
```

Reviewer: やっぱりここは `General Purpose` でもいいです

Author: なるほどー、 purpose がつかないと何に対して general なのか曖昧になってしまうんですね。

> It is a server that only returns HTTP 503 status (Service Unavailable).  

Reviewer: ```suggestion
This is a server that only returns HTTP 503 status codes (Service Unavailable).  
```

It is assumed to be used when downtime occurs in server maintenance.  

## Features

> - Responses `HTTP 503` for all request (without `/assets/*`).

Reviewer: ```suggestion
- Responds with `HTTP 503` for all requests (excluding `/assets/*`).
```

> - Responses `503.json` when contains `/json` in `Accept` header.

Reviewer: ```suggestion
- Responds with `503.json` when the `Accept` header contains `/json`.
```

> - Responses `503.json` when ends with `.json` for request path.

Reviewer: ```suggestion
- Responds with `503.json` when the request path ends with `.json`.
```

> - Otherwise it responses the contents of `503.html`.

Reviewer: ```suggestion
- Returns the contents of `503.html` for all other requests.
```

> - Supported `HTTPS`, supported multi domain.

Reviewer: ```suggestion
- Supports `HTTPS` and multiple domains.
```

> - Can specify Retry-After header by environment variable.

Reviewer: ```suggestion
- Allows specification of the `Retry-After` header through an environment variable.
```

## Usage

> ### Edit your contents

Reviewer: ```suggestion
### Customize response contents
```

Edit `503.html` and` 503.json` according to your use case.

#### Deploy assets 

If necessary, save the files used by `503.html` in the `assets` directory.

### Start server

> Listen port can be specified by the environment variable `PORT`.  

Reviewer: ```suggestion
The listen port can be specified by the environment variable `PORT`.  
```

> If not specified, it will listen on `80` port.  

Reviewer: ```suggestion
If not specified, it will listen on port `80`.  
```

> > Notice, on Linux etc, only root user can use 80 port.

Reviewer: ```suggestion
> Note that on Linux only the root user can use port 80.
```

```sh
PORT=8080 ./maint-server
```

### Listen HTTPS

> Use the following directory structure, if you want to use HTTPS.

Reviewer: ```suggestion
Use the following directory structure if you want to use HTTPS.
```

    └── ...
```

> - Supported multi domain.

Reviewer: ```suggestion
- Supports multiple domains
```

> - Certificate file must be a chained certificate combining intermediate certificates.

Reviewer: ```suggestion
- The certificate file must be a chained certificate combining intermediate certificates.
```

> - Determine the file by extension.

Reviewer: ```suggestion
- Files are determined by extension.
```

>     Save the certificate file extension `.crt` and the private key file extension` .key`.

Reviewer: ```suggestion
    Save the certificate file with extension `.crt` and the private key file with extension` .key`.
```

>   - Certificate for that domain will not be read if not have both.

Reviewer: ```suggestion
  - The certificate for that domain will not be read if it does not have both.
```

> - HTTPS listener will not start if there is no `ssl` directory or certificate file.

Reviewer: ```suggestion
- The HTTPS listener will not start if there is no `ssl` directory or certificate file.
```

#### Specify listen port

> Listen port can be specified by the environment variable `HTTPS_PORT`.

Reviewer: ```suggestion
The listen port can be specified by the environment variable `HTTPS_PORT`.
```

> If not specified, it will listen on `443` port.

Reviewer: ```suggestion
If not specified, it will listen on port `443`.
```

> > Notice, on Linux etc, only root user can use 443 port.

Reviewer: ```suggestion
> Note that on Linux, only the root user can use port 443.
```

```sh
PORT=8080 HTTPS_PORT=8443 ./maint-server
```

### Specify Retry-After

> Can include a `Retry-After` header in your response by specifying the date and time in the `RETRY_AFTER` environment variable in the form `yyyy-MM-dd hh:mm:ss+0000`.

Reviewer: ```suggestion
You can include a `Retry-After` header in your response by specifying the date and time in the `RETRY_AFTER` environment variable in the form `yyyy-MM-dd hh:mm:ss+0000`.
```
